integrationTests/mock/storageManagerStub.go

```diff
@@ -40,24 +36,6 @@ func (sms *StorageManagerStub) SetCheckpoint([]byte) {
 
 }
 
-// Prune --
-func (sms *StorageManagerStub) Prune([]byte, data.TriePruningIdentifier) {
-
-}
-
-// CancelPrune --
-func (sms *StorageManagerStub) CancelPrune([]byte, data.TriePruningIdentifier) {
-
-}
-
-// MarkForEviction --
-func (sms *StorageManagerStub) MarkForEviction(d []byte, m data.ModifiedHashes) error {
-	if sms.MarkForEvictionCalled != nil {
-		return sms.MarkForEvictionCalled(d, m)
-	}
-	return nil
-}
-
 // GetSnapshotThatContainsHash --
```

Comment:
maybe remove all occurrences of double - found in this file?

Reply:
Done.